charts/incubator/llama-gpt/values.yaml

```diff
@@ -0,0 +1,79 @@
+image:
+  repository: tccr.io/truecharts/llama-gpt-ui
+  pullPolicy: IfNotPresent
+  tag: v1.0.1@sha256:3e66190591adaf5bd91a0707c385bf0c26978c2627691659269335a105c93e7a
+apiImage:
+  repository: tccr.io/truecharts/llama-gpt-api
+  pullPolicy: IfNotPresent
+  tag: v1.0.1@sha256:39147d1e7829feb5c82e7654812ed629ee6b7a3422608597f64b232656fa8d83
+
+securityContext:
+  container:
+    runAsNonRoot: false
+    readOnlyRootFilesystem: false
+    runAsUser: 0
+    runAsGroup: 0
+    capabilities:
+      add:
+        - IPC_LOCK
```

Comment:
I think only the api needs to runs as root. UI can be rootless.
(Most likely user 1000 as its a node app)

Also note that IPC_LOCK is only needed when MLOCK is enabled.

And lastly 99% variable `PORT` can be used on the frontend to change its listenting port.

Reply:
how can we mutate this when MLOCK is enabled?

Reply:
I didnt see any variable called PORT documented.